contracts/gateway/GatewayBase.sol

```diff
@@ -95,12 +117,25 @@ contract GatewayBase is Organized {
     /** Bounty proposal block height. */
     uint256 public proposedBountyUnlockHeight;
 
+
+    /* Internal Variables */
+
+    /**
+     * Message box.
+     * @dev Keep this is at location 1, in case this is changed then update
+     *      constant MESSAGE_BOX_OFFSET accordingly.
+     */
+    MessageBus.MessageBox internal messageBox;
+
     /** Maps messageHash to the Message object. */
-    mapping(bytes32 => MessageBus.Message) messages;
+    mapping(bytes32 => MessageBus.Message) internal messages;
```

Comment:
Can this be `public`? Any issue if it will be public?
In one of the other open PR, its changed to public